.github/workflows/broker.yml

```diff
@@ -98,20 +98,36 @@ jobs:
     if: github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/')
     steps:
       - uses: actions/checkout@v2.3.4
+      - name: Setup qemu
+        uses: docker/setup-qemu-action@v1.2.0
```

Comment:
Let's install only the required platforms with `platforms` attribute. https://github.com/docker/setup-qemu-action#inputs